boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig

```diff
@@ -35,3 +35,5 @@ CONFIG_ADC_ITE_IT8XXX2=y
 CONFIG_SOC_FLASH_ITE_IT8XXX2=y
 CONFIG_PWM=y
 CONFIG_PWM_ITE_IT8XXX2=y
+CONFIG_SENSOR=y
+CONFIG_TACH_IT8XXX2=y
```

Comment:
Ack. I put the conditional in the soc Kconfig.defconfig.series, because our Kconfig.defconfig sources from  Kconfig.defconfig.series.

Ok, I will create aonther PR for the rest.

Reply:
Still need to remove `CONFIG_SENSOR=y` from `boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig`

Reply:
We want to enable CONFIG_SENSOR on it8xxx2_evb board, if we remove CONFIG_SENSOR=y in boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig, then where shall I enable it?

Reply:
Done